### ISSUE TITLE
Owner Op incorrect

### ISSUE THREAD

**Issue by bhetherman** [flexflow/FlexFlow#59]
Tensor t = ff.conv2d(input, out_channels, 1, 1, 1, 1, 0, 0, AC_MODE_NONE); //owner is null
t = ff.batch_norm(t, false); //owner is Conv2d_11_100
t = ff.relu(t); //owner is Conv2d_11_100 but should be batch norm

Any idea why the owner ops are not propagating to the model layers correctly? It seems to be something with batch_norm because all the other layers are working?

**Comment by jiazhihao** [comment]
The issue should have been fixed. @bhetherman Let me know if you still observe the same error and I will reopen this.
